modules/3d-tiles/src/lib/parsers/parse-3d-tile-header.ts

```diff
@@ -94,13 +111,12 @@ export function normalizeTileHeaders(tileset) {
  * TODO Check if Tile3D class can be a return type here.
  * @param tileset
  */
-export async function normalizeImplicitTileHeaders(tileset: Tileset3D) {
-  if (!tileset.root) {
-    return null;
-  }
-
+export async function normalizeImplicitTileHeaders(
```

Comment:
Return types seem to be missing everywhere, maybe make a pass in a separate PR?

Reply:
Yes, will create another PR